tools/lib/provision.py

```diff
@@ -126,6 +126,7 @@
     "gettext",              # Used by makemessages i18n
     "curl",                 # Used for fetching PhantomJS as wget occasionally fails on redirects
     "netcat",               # Used for flushing memcached
+    "libcurl4-openssl-dev",  # Thumbor Required Library
```

Comment:
Why is this list of libraries so much shorter than the one in `puppet`?  Makes me wonder if we're missing some that should be here.